### ISSUE TITLE
Annotations show up in AST-Files after new build

### ISSUE THREAD

**Issue by dpeuscher** [seba--/sugarj#50]
When changing something the depending AST-Files are automatically updated but not with the same method as they show annotations which generally are not shown.

**Comment by seba--** [comment]
You mean the ".aterm" files? They update already when editing a source file. Can you clarify?

**Comment by dpeuscher** [comment]
Yes, when they update upon editing a source they show annotations on every node. These are not shown if before the editing nor if you reopen the aterm (via the transform-button)

**Comment by seba--** [comment]
I cannot reproduce the problem. Can you confirm this still happens in the latest release of SugarJ? How can I reproduce it?

**Comment by dpeuscher** [comment]
I recorded it for you: http://www.youtube.com/watch?v=kTdYtrUMtA8
